.github/workflows/kfp-backend-postgres-tests.yml

```diff
@@ -0,0 +1,245 @@
+name: KFP Backend Postgres Tests
+
+on:
+  workflow_dispatch:
+    inputs:
+      test_repository:
+        description: "owner/repo to checkout (empty = upstream)"
+        required: false
+        default: ""
+      test_ref:
+        description: "Git ref (branch/tag/SHA) to test"
+        required: true
+        default: "master"
+      test_paths:
+        description: "Go package paths"
+        required: true
+        default: "./backend/test/initialization"
+
+jobs:
+  postgres-pgx:
+    name: Backend Postgres (pgx via Kustomize+KinD)
+    runs-on: ubuntu-latest
+    continue-on-error: true # Temporarily allow failures; switch to false after LargeText/storage changes land
+
+    steps:
+      - name: Checkout target code
+        uses: actions/checkout@v4
+        with:
+          repository: ${{ github.event.inputs.test_repository || github.repository }}
+          ref: ${{ github.event.inputs.test_ref }}
```

Comment:
I don't think we need this, if you are running the action from a fork it will automatically check out code from there (hence `github.repository`), and there's no reason to run this CI from a remote repository in the KFP repo.